src/api/v1/statuses.ts

```diff
@@ -793,15 +852,36 @@ app.post(
       where: eq(posts.id, id),
       with: getPostRelations(owner.id),
     });
+    const activity = toAnnounce(post!, fedCtx);
     await fedCtx.sendActivity(
       { username: owner.handle },
       "followers",
-      toAnnounce(post!, fedCtx),
+      activity,
       {
         preferSharedInbox: true,
         excludeBaseUris: [new URL(c.req.url)],
       },
     );
+    if (!owner.account.protected) {
+      const acceptedRelays = await db.query.relays.findMany({
+        where: eq(relays.state, "accepted"),
+        with: {
+          relayServerActor: true,
+        },
+      });
+      await fedCtx.sendActivity(
+        owner,
+        acceptedRelays.map((relay) => ({
+          id: new URL(relay.relayServerActor.iri),
+          inboxId: new URL(relay.relayServerActor.inboxUrl),
+        })),
+        activity,
+        {
+          preferSharedInbox: true,
```

Comment:
We don't have to turn this on as we aren't providing recipients' shared inbox URLs here.